BibBuddy/app/src/main/java/de/bibbuddy/NoteRecyclerViewAdapter.java

```diff
@@ -25,10 +25,10 @@
 import java.util.Locale;
 
 /**
- * NoteRecyclerViewAdapter provides a binding from the noteList to the view
- * that is displayed within the RecyclerView of the NotesFragment.
+ * NoteRecyclerViewAdapter provides a binding from a note-list to a corresponding
+ * RecyclerView-list.
```

Comment:
As far as I now, in this case in the english language no "-" would be used :)
```suggestion
 * NoteRecyclerViewAdapter provides a binding from a note list to a corresponding
 * RecyclerView list.
```